Facepunch.Steamworks/Structs/UgcItem.cs

```diff
@@ -236,6 +239,56 @@ public float DownloadAmount
 			}
 		}
 
+		/// <summary>
+		/// Queries all favorite items (of current user) looking for this one.
+		/// </summary>
+		public async Task<bool> IsFavoriteAsync()
```

Comment:
Should maybe be a TaskValue instead of a Task?

Reply:
ValueTask doesn't work in Unity using netstandard